src/platform/user/tests/authentication/components/loginActions.unit.spec.js

```diff
@@ -2,13 +2,23 @@ import React from 'react';
 import { expect } from 'chai';
 import { mount } from 'enzyme';
 import sinon from 'sinon';
-
+import { Provider } from 'react-redux';
 import * as authUtilities from '../../../authentication/utilities';
 import LoginActions from '../../../authentication/components/LoginActions';
 import { CSP_IDS } from '../../../authentication/constants';
 
 describe('login DOM ', () => {
   const sandbox = sinon.createSandbox();
+  const mockStore = {
+    getState: () => ({
+      featureToggles: {
+        // eslint-disable-next-line camelcase
+        showsignInPageAndModalExperiment: true,
```

Comment:
1. Create tests so that you can toggle between the 2 states and confirm your modal shows appropriately.
2. Create another test to loop through all of the external applications of USiP and confirm that they do not show the modal.